src/App.js

```diff
@@ -5,9 +5,7 @@ import MainMenu from './main-menu.js';
 import './App.css';
 import AdditionModule from './addition/addition-module.js';
 import SubtractionModule from './subtraction/subtraction-module.js';
-import { getStudents } from './helpers/database-helpers.js';
-import { getStudentByName } from './helpers/database-helpers.js';
-import { checkLogin } from './helpers/database-helpers.js';
+import { checkLogin,insertPerson,getStudentByName,insertScore,insertClass,insertObject } from './helpers/database-helpers.js';
```

Comment:
small nitpick, we should have spaces after each comma.
{ checkLogin, insertPerson, ... }